docs/user-guides/manual-setup.md

```diff
@@ -0,0 +1,162 @@
+# Partner chains setup
+
+Partner Chains provides text based CLIs, for setting up a partner chain and starting nodes, called wizards.
+In the [chain builder](./chain-builder.md), [permissioned node owner](./permissioned.md), and [registered node owner](./registered.md) guides these wizards are used.
+
+This document explains what is performed by those wizards and how to perform such operations manually.
+
+For dependency setup please see the respective guide for your desired setup.
+
+## Generating keys
+
+`<node-executable> wizards generate-keys` creates 3 specific keys, stores them in the keystore and also creates a JSON file with public keys.
```

Comment:
Instead of always writing `<node-executable>` I would suggest to just introduce `pc-node` as the node executable. That also makes all the invocations below look a bit less alien because we are actually looking at commands that are to be run in a shell.